src/SignalR/clients/ts/signalr/src/HubConnection.ts

```diff
@@ -180,9 +180,9 @@ export class HubConnection {
             await this._startInternal();
 
             if (Platform.isBrowser) {
```

Comment:
It looks like our `Platform.isBrowser` check is broken. It's currently `return typeof window === "object"` which is always `true` for React Native even outside the browser. I just tested this on my Android phone.

When running React Native inside the browser, you can still reference `document` directly, and I think we still do want to attach the "freeze" event listener in that case.

I think changing `Platform.isBrowser` to `return typeof document === "object"` might be the better fix.